css-values/Overview.bs

```diff
@@ -2287,4 +2651,10 @@ Security and Privacy Considerations</h2>
 	This specification defines units that expose the user's screen size
 	and default font size,
 	but both are trivially observable from JS,
-	so they do not constitutate a new privacy risk.
```

Comment:
Starting to add new considerations for proposed units. Most of them are just as safe as existing ones.